test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -1821,7 +1823,9 @@ BOOST_AUTO_TEST_CASE(warn_var_from_zero)
 			}
 		}
 	)";
-	CHECK_WARNING(sourceCode, "uint8, which can hold values between 0 and 255");
```

Comment:
It should be checked that this error message is present until we disallow var completely.

Reply:
Can you clarify? `CHECK_WARNING_ALLOW_MULTI` seems to only allow the first warning to be checked, so if `var` is used in this test it will cause the first warning and it will not be possible to check for the size warning. Is there a way to check both?

Reply:
I fear someone has to extend the macro...

Reply:
I could look into this within the next week or so. Do you want to file it as a separate issue?